### PR TITLE
324 improve the pipeline description

### PR DESCRIPTION
**Pull request by mhkc**
# Description

This PR reworks the pipeline documentation. 

## Primary function of PR
- [ ] Hotfix
- [x] Minor functionality improvement
- [ ] Major functionality improvement / New type of analysis
- [ ] Backward-breaking functionality

# Testing
_Either describe a procedure, or add data; that confirms that the PR resolves what it sets out to do_

# Sign-offs
- [ ] Code reviewed by @octocat
- [ ] Code tested by @octocat

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,223 +9,32 @@ _Just Another System for Epityping NGS data_
...
-#### Update MLST database
-
-```
-bash /path/to/jasen/assets/mlst_db/update_mlst_db.sh
-```
+The pipeline currently support a small set of microbiota and the support are in different stages of development. See the documentation of information on the supported analysis for each species and what the development status means.
 
+| Species                      | Development status |
+|------------------------------|--------------------|
+| *Staphylococcus arueus*      | Draft              |
```
Comment (svartapaerlan) [inline README.md:18]: Do you reckon this is true? Should we not say beta?

File: docs/source/install.md [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+## Create personalised TBProfiler database
+
+### Install jasentool
+
+```bash
+git clone git@github.com:ryanjameskennedy/jasentool.git && cd jasentool
+pip install .
+```
+
+### Create input csv that is used as tbdb input (composed of FoHM, WHO & tbdb variants)
```
Comment (svartapaerlan) [inline docs/source/install.md:126]: Remove from here...

File: docs/source/install.md [hunk 3]
```diff
@@ -0,0 +1,145 @@
...
+jasentool converge --output_dir /path/to/jasen/assets/tbdb
+```
+
+### Create tbdb (ensure tb-profiler is installed)
+
+```bash
+cd /path/to/jasen/assets/tbdb
+tb-profiler create_db --prefix converged_who_fohm_tbdb
+tb-profiler load_library converged_who_fohm_tbdb
+```
```
Comment (svartapaerlan) [inline docs/source/install.md:138]: To here...

File: docs/source/overview.md [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+:alt: jasen flowchart
+
+Flowchart depiciting the main workflows, its output, and processes the workflow are constituted of. The colour of the line denotes the workflow and the area with yellow background indicate quality control processes.
+```
+
+## Status on workflow development
+
+Jasen is being continiously developed with the aim to add support for more species. Therefore can workflows be at different stages of development and evolve at different pace. A workflow can be either of three stages (*development*, *draft*, and *stable*) of development, these reflect the stability and the likelyhood of radical canges being made to the workflow.
+
+Workflows under **development** are being actively worked at. While all workflows should be functional in the *masters* branch and for every release, there are no guarantee that workflows under development produce accurate results and major changes can be introduce without notice. Use a your own descression.
```
Comment (svartapaerlan) [inline docs/source/overview.md:17]:
```suggestion
Workflows under **development** are being actively worked at. While all workflows should be functional in the *masters* branch and for every release, there are no guarantee that workflows under development produce accurate results and major changes can be introduce without notice. Use a your own discretion.
```

File: docs/source/overview.md [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+
+**Draft** workflows are considered feature complete and in the process of being validated. Software versions can still change without notice. Use with descression.
```
Comment (svartapaerlan) [inline docs/source/overview.md:19]:
```suggestion
**Draft** workflows are considered feature complete and in the process of being validated. Software versions can still change without notice. Use with discretion.
```
```suggestion
**Draft** workflows are considered feature complete and in the process of being validated. Software versions can still change without notice. Use with discretion.
```

File: docs/source/install.md [hunk 6]
```diff
@@ -0,0 +1,145 @@
...
+
+### Create singularity images. 
+
+Note: The containers that need to be built locally require sudo privileges.
+
+```bash
+cd container
+sudo make build_local_containers
+make download_remote_containers
```
Comment (svartapaerlan) [inline docs/source/install.md:36]: I think you should check the current documentation for this

File: README.md (hunk 1 above)
Comment (mhkc) [reply README.md:18]: The exact nomenclature doesnt matter to me. But I think draft is perhaps more expressive than beta. We also define the difference between the different statuses in the text as well.

File: docs/source/overview.md [hunk 7]
```diff
@@ -0,0 +1,41 @@
...
+|---------------------------------------------------------------------------|--------------------|
+| [*Staphylococcus arueus*](./workflows/staphylococcus_aureus.md)           | Draft              |
+| [*Escherichia coli*](./workflows/escherichia_coli.md)                     | Draft              |
+| [*Mycobacterium tuberculosis*](./workflows/mycobacterium_tuberculosis.md) | Draft              |
+
+## Input
+
+Jasen is designed for analysis of sequenced whole genomes from bacterial isolates. The pipeline takes short-read sequenced reads, in FastQ format, from Illumina and Ion Torrent as input.
+
+The paths to the reads are defined in a CSV file with sample id, and optionally sequencing run and lims id. See [the usage](./useage.md) page for information on how to start a analysis.
```
Comment (svartapaerlan) [inline docs/source/overview.md:35]: Useage is spelt incorrectly

File: docs/source/useage.md [hunk 8]
```diff
@@ -0,0 +1,29 @@
+# Usage
```
Comment (svartapaerlan) [inline docs/source/useage.md:1]: This file's name is incorrectly spelt

**Review by svartapaerlan — APPROVED** [review]
LGTM!

**Pull request merged**